SimPEG/regularization/pgi.py

```diff
@@ -764,11 +756,14 @@ def alpha_pgi(self):
 
     @alpha_pgi.setter
     def alpha_pgi(self, value):
-        if isinstance(value, (float, int)) and value < 0:
```

Comment:
Just use the `validate_float` function from `SimPEG.utils`

Reply:
It's up to you to decide if it makes sense to allow passing `None` type to this variable. It seems like it'd be better off just leaving it's default (set during `__init__`) at `1.0` and the not allowing a None type to be set to `alpha_pgi`

Reply:
Let be explicit and not allow None.